File: PlayLib-Back/PlayLib-Back/PlayLib.Data/Entities/Videogame.cs
using PlayLib.Data.Entities;

public class Videogame {
    public Guid Id { get; set; }
    public string Name { get; set; } = null!;
    public string Description { get; set; } = null!;
    public string Developer { get; set; } = null!;
    public string? ImageRoute { get; set; }
    public DateTime? ReleaseDate { get; set; }
    public string Format { get; set; } = null!;

    public ICollection<Review> Reviews { get; set; } = new List<Review>();
    public ICollection<VideogameLibrary> Libraries { get; set; } = new List<VideogameLibrary>();
    public ICollection<FavouriteVideogame> Favourites { get; set; } = new List<FavouriteVideogame>();

    public ICollection<LanguageVideogame> Languages { get; set; } = new List<LanguageVideogame>();
    public ICollection<TagVideogame> Tags { get; set; } = new List<TagVideogame>();
}

File: PlayLib-Front/src/app/model/videogame.ts
import { Tag } from "./tag";
import { Language } from "./language";

export interface Videogame {
  id: string;
  name: string;
  description: string;
  developer: string;
  image_route: string;
  release_date: Date;
  format: string;
  state: string;
  tags: Tag[];
  languages: Language[];
  isInLibrary: boolean;
  isFavourite: boolean;
}